qgis-app/static/jquery-ratings/jquery.ratings.1.0.js

```diff
@@ -42,8 +42,21 @@ jQuery.fn.ratings = function(stars, initialRating) {
       star.addClass('jquery-ratings-star');
       
       //Add the full css class if the star is beneath the initial rating.
+      // The last star will depend on the decimal left
       if(starIdx < initialRating) {
-        star.addClass('jquery-ratings-full');
+        let ratingDiff = initialRating - starIdx
+        if (ratingDiff > 0 && ratingDiff <= 0.3) {
+          star.addClass('jquery-ratings-quarter');
+        }
+        else if (ratingDiff > 0.3 && ratingDiff <= 0.6) {
+          star.addClass('jquery-ratings-half');
+        }
+        else if (ratingDiff > 0.6 && ratingDiff <= 0.9) {
+          star.addClass('jquery-ratings-half-and-quarter');
+        }
+        else {
+          star.addClass('jquery-ratings-full');
+        }

```

Comment:
Cool, thanks 👍 On a first sight I was wondering about the uneven distribution >0.3 <= 0.6 for the "half star", but how you implemented it, every "partial star" is displayed for a range of 0.3 which is fine. With the stars it's hard to spot how many decimals are represented anyway :-)